### ISSUE TITLE
Améliorer module_group en backend et bdd

### ISSUE THREAD

**Issue by JulienCorny** [PnX-SI/GeoNature#2502]
voir #2333 pour le contexte

- créer table gn_commons.bib_module_groups
- renommer module_group de gn_commons.t_modules en id_module_group, clé étrangère vers gn_commons.bib_module_groups

**Comment by andriacap** [comment]
Salut !

Je me permet de reprendre un commentaire de l'issue #2333 pour soulever quelques questions.
J'aurais une question concernant la table `bib_module_groups` à créer . A part le champ d' `id_module_group` est ce qu'on attend d'avoir d'autre champ ( type : `label_group` ou `num_group` , `couleur`  ou encore doit elle service de table d'association qui pointerait vers une autre table ? ) 

Je pose ces questions car il  été mentonné qu'il serait intéressant de regrouper les modules par couleur et donc savoir comment organiser le modèle de données .

Merci d'avance pour vos retours !  :) 

@JulienCorny  n'étant pas là cette semaine je me permet de mentionner d'autres personnes ayant participées à la rélfexion de l'issue #2333 .
@camillemonchicourt  , @TheoLechemia 

> OK bien vu. Je n'avais plus en tête ce champs `module_group`, certainement par le fait qu'il n'a pas d'usage actuellement, à ma connaissance. On avait certainement du l'ajouter avec cette idée de pouvoir regrouper des modules.
> 
> Si on le laisse en champs texte comme ça, ça peut fonctionner, mais c'est fragile et limité. Cela impliquerai que ce champs soit bien saisi de manière similaire pour tous les modules d'un groupe. Et on ne pourrait modifier facilement le label du groupe, ni lui définir un picto par exemple.
> 
> Une table `gn_commons. bib_module_groups` et un clé étrangère `id_module_group` dans la table `t_modules` éviterait ces limites.
> 
> PS : Depuis la version 2.12.0, il est possible de modifier les infos d'affichage des modules directement dans le module "Admin" pour permettre à des administrateurs de modifier facilement le label, le picto, l'ordre ou encore l'URL de la doc d'un module sans avoir accès à la BDD : ![image](https://user-images.githubusercontent.com/4418840/230070223-f4884d3e-dfe3-4322-87dd-5671fd8dadf6.png)

**Comment by camillemonchicourt** [comment]
Oui, il faut pouvoir donner un nom au groupe, et en effet un ordre. Voir la table `t_modules` pour avoir un nommage cohérent des champs.
La couleur je ne pense pas que cela soit un usage à retenir.
Il faut bien gérer le cas où un module n'est pas associé à un groupe, qui doit rester un usage optionnel.
